pkg/virt-api/webhooks/validating-webhook/admitters/vmi-create-admitter_test.go

```diff
@@ -3813,6 +3797,123 @@ var _ = Describe("Validating VMICreate Admitter", func() {
 			Entry("explicit only", useExplicitHyperV, doNotUseHyperVPassthrough, true),
 			Entry("no hyperv use", doNotUseExplicitHyperV, doNotUseHyperVPassthrough, true),
 		)
+
+	})
+
+	Context("Watchdog device validation", func() {
+		var vmi *v1.VirtualMachineInstance
+
+		BeforeEach(func() {
+			vmi = api.NewMinimalVMI("testvmi")
+		})
+
+		DescribeTable("validate for amd64",
+			func(setup func(), expectedMessage string, shouldReject bool) {
+				setup()
```

Comment:
Why is this a function instead of a `*v1.Watchdog`? All the setup does is setting the watchdog device anyway?